About.html

```diff
@@ -0,0 +1,159 @@
+    </nav>
+</header>
+    <main>
+        <!-- section of summit Introduction -->
+        <section class="intro pb-5 pt-5 big-bg" id="home">
+            <div class="hamburger">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </div>
+            <div class="d-flex flex-column">
+                <div class="mx-5">
+                    <h2 class="c-2 fs-2">"Hello! Globalized World"</h2>
+                    <h1 class="intro-title">NATIONAL EDUCATION SUMMIT
+                        <br>2023
+                    </h1>
+                </div>
+                <div class="intro-text">
+                    <p class="lead mx-3 px-2 py-4">
+                        Explore the comprehensive program on offer in Melbourne including Conferences, 
+                        Seminars and Workshops. Over the two days educators can attend Diverse Leaners Symposium, 
+                        Wellbeing for Future Focused Schools, Capacity Building School Libraries, 
+                        STEM Across the Curriculum, Literacy Masterclass, Classroom of the Future and Free Seminars.
+                    </p>
+                </div>
+                <h2 class="about-contact text-center mx-3">Please contact us per Email for any furthet questions <br>about National Eductional Summit</h2>
+                <h3 class="abou-email text-center mx-3">educationvisitor@iecgroup.com.au</h3>
+            </div>
+        </section>
+        
+        <!-- Summit logo section-->
+        <section class="border" id="logo">
+            <h2 class="logo-text text-center py-5">The National Educational Summit<br> Logo 2023</h2>
+            <div class="br-line"></div>
+            <h2 class="about-contact text-center mx-3">The logo of National Educational Summit was decided<br>through recently held logo competition</h2>
+            <div>
+                <img class="NES-logo pb-5" src="./images/NES+LOGO.png" alt="NES+LOGO">
+            </div>
+        </section>
+
+        <!--Section of Previous Educational Summits -->
+        <section id="past">
+            <h2 class="logo-text text-center pt-5">See the Past National Educational Summits</h2>
+            <div class="br-line"></div>
+            <h2 class="about-contact text-center mx-3">Take a Look at the last two National<br>Educational Summits</h2>
+            <div class="past-c">
+                <div class="past-1 mx-3 mt-5">
+                <h2 class="about-past0 text-center mx-3">2022</h2>
+                <h2 class="about-past1 text-center mx-3">National Educational Summit 2022, Washington,DC.</h2>
+                </div>
+                <div class="past-2 mx-3 mt-5">
+                    <h2 class="about-past3 text-center mx-3">2021</h2>
+                    <h2 class="about-past4 text-center mx-3">National Educational Summit 2021(online), Washington,DC.</h2>
+                </div>
+            </div>
+        </section>
```

Comment:
- [ ] The previous reviewer should have asked for this change. However, he didn't. So it's optional but highly recommended to use the same color for texts as in the given design:
yours: 
![image](https://user-images.githubusercontent.com/91486791/235118771-44af9f55-31eb-4ee9-ac91-977a73e436bb.png)
![image](https://user-images.githubusercontent.com/91486791/235119257-5dd28b30-b9c1-407a-9078-9b5d65c056a0.png)

given design:
![image](https://user-images.githubusercontent.com/91486791/235119015-f3be75b0-0866-4280-b83e-d078e6cc7c48.png)
![image](https://user-images.githubusercontent.com/91486791/235119323-80ace26e-7954-48fc-b4f3-3964ab73f865.png)